src/Commands/Presets/UICommand.php

```diff
@@ -57,6 +57,7 @@ public static function clean_up()
     {
         File::delete(base_path('postcss.config.js'));
         File::delete(base_path('tailwind.config.js'));
+        File::delete(resource_path('views/scss/app.css'));
```

Comment:
Hey Marco, can you fix the file path here. The app.css file is inside resources/scss not in views.

Reply:
My bad! should be ok now|!